### PR TITLE
Added Lime & Green

### PR DESCRIPTION
**Pull request by PrasenjitM9**
these are very natural colors by pm9

### CODE REVIEW

**Review by prasenjitmking — APPROVED** [review]
pls check it once bcz you are the contributor

**Comment by PrasenjitM9** [comment]
Ok! thanks for intimating me, my job profile!

File: poem.txt [hunk 1]
```diff
@@ -17,3 +17,6 @@ Red
 Thanks for the colors of the rainbow.
 Add the unicors.
 Pony
+
+Lime
```
Comment (PrasenjitM9) [inline poem.txt:21]: Green has many variety.. so can suggest me to which is better color...

**Review by PrasenjitM9 — COMMENTED** [review]
pls sir check contributor review!

File: poem.txt (hunk 1 above)
Comment (prasenjitmking) [reply poem.txt:21]: Forest Green is best one.

Comment (PrasenjitM9) [reply poem.txt:21]: It's ok I will add forest green!!

**Comment by PrasenjitM9** [comment]
Please close these conversation!!

**Pull request closed without merging**